### ISSUE TITLE
HI

### ISSUE THREAD

**Issue by songheewon** [KimmyJay/practice_problems#1]
HI FRANCIS (╹ڡ╹ )

**Comment by KimmyJay** [comment]
HI MS. SAMSONG (☞ﾟヮﾟ)☞

**Comment by songheewon** [comment]
SPARTA GOOD? PROJECT DONE? ヾ(⌐■_■)ノ♪

**Comment by KimmyJay** [comment]
JUST FINISH AND SADARI! ∩(˵■‿■˵)つ¤=[]:::::> U START PROJECT YET?

**Comment by songheewon** [comment]
I'M LEARNING JAVA o(TヘTo) 새로운 팀원은 GOOD?!

**Comment by KimmyJay** [comment]
LOL NEW TEAMMATE IS GHOST ¯\(◉◡◔)/¯ I THINK HE HAS ISSUES...

**Comment by songheewon** [comment]
WHY.....? ╚(•⌂•)╝ THEN, KYUMIN IS UPSET ?

**Comment by KimmyJay** [comment]
YES... KYUMIN SENPAI SUPER MAD AND COMING FOR YOU! (pﾟ ﾛﾟ)==p)`д)

**Comment by songheewon** [comment]
KYUMIN SENPAI..... CALM DOWN ┌( ಠ_ಠ)┘

**Comment by KimmyJay** [comment]
BE CAREFUL!! (」゜ロ゜)」HOW IS STUDY LIFE BALANCE AT SSAFY??

**Comment by Roshu2** [comment]
╰（‵□′）╯

**Comment by songheewon** [comment]
SSAFY... HARD TO WAKE UP.......

**Comment by songheewon** [comment]
KYUMIN HI~~~~\(@^0^@)/

**Comment by KimmyJay** [comment]
MS. SONG PLS GIVE FEEDBACK OK?
[이거대여. 피드백](https://docs.google.com/forms/d/1oGm5Uo0P7pfQb989DdF7dqR1bGxmsi9Xb7LkCljVYI8/edit)

**Comment by songheewon** [comment]
OK FRANCIS!

**Comment by KimmyJay** [comment]
U GET CHICKEN!

**Comment by songheewon** [comment]
THANK YOU VERY MUCH!!!!!!!!!!!!!

**Comment by KimmyJay** [comment]
HI WONSONG (╹ڡ╹ )

**Comment by songheewon** [comment]
HI FRANCIS ٩(๑❛ᴗ❛๑)۶

**Comment by KimmyJay** [comment]
SSAPY GOOD? HOW ARE YOU? ヾ(⌐■_■)ノ♪

**Comment by songheewon** [comment]
NICE! HOW ABOUT YOU?!

**Comment by KimmyJay** [comment]
I'M WORKING AT STARTUP o(TヘTo) 스프링 굳??

**Comment by songheewon** [comment]
OH YOU ARE WORKING!!! WHERE??? SPRING... I CANT UNDERSTAND... (⊙_⊙)？

**Comment by KimmyJay** [comment]
I am working at startup called miso doing data stuff with sql and pandas ㅠㅠ How is ssapy and java ㅇ_ㅇ
